src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/tests/test_webapp_commands.py

```diff
@@ -155,6 +152,50 @@ def test_create_in_different_group(self, resource_group, resource_group_location
         ])
 
 
+# Test Framework is not able to handle binary file format, hence, only run live
+class AppServiceLogTest(LiveScenarioTest):
+    @ResourceGroupPreparer()
+    def test_download_win_web_log(self, resource_group):
+        import zipfile
+        from pathlib import Path
+        webapp_name = self.create_random_name(prefix='webapp-win-log', length=24)
+        plan = self.create_random_name(prefix='win-log', length=24)
+        self.cmd('appservice plan create -g {} -n {}'.format(resource_group, plan))
+        self.cmd('webapp create -g {} -n {} --plan {} --deployment-source-url https://github.com/yugangw-msft/azure-site-test.git -r "node|6.1"'.format(resource_group, webapp_name, plan))
+        time.sleep(30)  # 30 seconds should be enough for the deployment finished(Skipped under playback mode)
+
+        # sanity check the traces
+        _, log_file = tempfile.mkstemp()
+        log_dir = log_file + '-dir'
+        log_file = log_file.replace('\\', '\\\\')
+        self.cmd('webapp log download -g {} -n {} --log-file {}'.format(resource_group, webapp_name, log_file))
+        zip_ref = zipfile.ZipFile(log_file, 'r')
+        zip_ref.extractall(log_dir)
+        p = Path(log_dir, r'LogFiles\kudu\trace')
+        self.assertTrue(p.is_dir())
```

Comment:
Can this be done by `os.path.isdir(os.path.join(log_dir, 'LogFiles', 'kudu', 'trace'))`?

It's preferred to:
1. Avoid platform-specific separator (`\` vs `/`)
2. Use `os.path`

Reply:
Thanks. I was a bit sloppy on the test code.)